selfservice/strategy/oidc/strategy.go

```diff
@@ -447,11 +451,13 @@ func (s *Strategy) HandleCallback(w http.ResponseWriter, r *http.Request, ps htt
 		}
 	}
 
-	if err := claims.Validate(); err != nil {
+	if err = claims.Validate(); err != nil {
 		s.forwardError(w, r, req, s.handleError(w, r, req, pid, nil, err))
 		return
 	}
 
+	span.SetAttributes(attribute.StringSlice("claims", maps.Keys(claims.RawClaims)))
```

Comment:
Wouldn't this leak PII into the tracing system?

Reply:
No, this is just the keys, meaning the claim keys. The PII is just in the values, which we don't send.